services/motion/localizer.go

```diff
@@ -93,7 +93,7 @@ func (m *movementSensorLocalizer) CurrentPosition(ctx context.Context) (*referen
 	}
 
 	pose := spatialmath.NewPose(spatialmath.GeoPointToPoint(gp, m.origin), o)
-	return referenceframe.NewPoseInFrame(m.Name().ShortName(), spatialmath.Compose(pose, m.calibration)), nil
+	return referenceframe.NewPoseInFrame(referenceframe.World, spatialmath.Compose(pose, m.calibration)), nil
```

Comment:
why is this changing, shouldn't it remain as with respect to the localizer?, i.e. the observer?

Reply:
Having this report with respect to the observer actually causes a lot of problems.

Firstly, it's wrong. If you have a camera computing its own pose, say (X: 1000, Y:2000), then that's *not* a pose in the frame of the camera. That's the pose of the camera in the frame of something else. Saying `PoseInFrame("camera",  (X: 1000, Y:2000))` instead refers to a point ~3 meters away from the camera.

Secondly, the purpose of a localizer is to locate something with regard to some world frame, whether that is the actual world (gps) or a slam map.

But the first reason is the main one. Prior to this PR we were simply pulling the pose out of the PiF and assigning it to world anyway. Because the actual thing we're returning breaks the math, when I added the code to actually properly transform the PiFs into the desired frame, lots of things broke due to the inverted transform.